doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -271,27 +271,27 @@ the possible following elements delimited by single quotes:
 
```

Comment:
I suppose this refers to tokens such as `'else  '` with spaces between the apostrophes?
Maybe move the bullet point to the end of the list and say "extra spaces in other tokens are preserved in the output"?